app/models/ca_search_forms.php

```diff
@@ -996,6 +996,44 @@ public function getAvailableBundles($pm_table_name_or_num=null, $pa_options=null
 			);
 		}
 
+		// Hidden Field
+		$va_additional_settings = array(
+			'hidden_field_name' => array(
+				'formatType' => FT_TEXT,
+				'displayType' => DT_FIELD,
+				'width' => 25, 'height' => 1,
+				'takesLocale' => false,
+				'required' => true,
+				'label' => _t('Field Name'),
+				'description' => _t('Name of the hidden input field. For example <em>ca_objects.type_id</em>.'),
+			),
+			'hidden_field_value' => array(
+				'formatType' => FT_TEXT,
+				'displayType' => DT_FIELD,
+				'width' => 25, 'height' => 1,
+				'takesLocale' => false,
+				'requireValue' => true,
+				'label' => _t('Value'),
+				'description' => _t('Value of the hidden input field.')
+			)
+		);
+		$t_placement->setSettingDefinitionsForPlacement($va_additional_settings);
+		$vs_bundle = 'hidden_field';
+		$vs_display = "<div id='searchFormEditor_$vs_bundle'><span class='bundleDisplayEditorPlacementListItemTitle'>"._t('Custom').'</span> '.($vs_label = _t('Hidden Field'))."</div>";
```

Comment:
Not sure whether the class name `bundleDisplayEditorPlacementListItemTitle` is correct?